src/runtime/runtime_libc.go

```diff
@@ -0,0 +1,15 @@
+// +build baremetal
+
+package runtime
+
+import "unsafe"
+
+//go:export malloc
```

Comment:
The typical Go version is `//export`, not `//go:export`. TinyGo treats those identically, but I think it's best to move to `//export` everywhere (and not introduce new `//go:export` pragmas) to be consistent with the wider Go ecosystem.

The same goes for `libc_free` below.

Reply:
Done